src/main/java/Spaceman.java

```diff
@@ -0,0 +1,98 @@
+import java.util.Scanner;
+
+public class Spaceman {
+    public static void main(String[] args) {
+        String logo = "  ____  _____   ___    _____ _____ __    __   ___   __   __\n"
```

Comment:
```suggestion
        String LOGO = "  ____  _____   ___    _____ _____ __    __   ___   __   __\n"
```
Constants are recommended to be in ALL CAPS

Reply:
Thanks for pointing out!